tests/ragger/test_error_cmd.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from ragger.error import ExceptionRAPDU
+
+from boilerplate_client.exception import *
+
+
+@pytest.mark.xfail(raises=ExceptionRAPDU)
```

Comment:
Can we check that the error is a ClaNotSupportedError?

Reply:
To be updated in another PR.

Reply:
We could at least expect `BoilerplateError` instead of `ExceptionRAPDU`)

Reply:
Except it does not work either, as we are using the backend to speak with the app directly here, not the boilerplate client, so it's pure `Ragger` territory here.